src/test/ui/rfc-2632-const-trait-impl/const-drop.rs

```diff
@@ -0,0 +1,23 @@
+#![feature(const_trait_impl)]
+#![feature(const_fn_trait_bound)]
+#![feature(const_mut_refs)]
+#![feature(const_panic)]
+
+struct S;
+
+impl const Drop for S {
+    fn drop(&mut self) {
+        // NB: There is no way to tell that a const destructor is ran,
```

Comment:
you can give S a field that is a mutable borrow to something and then mutate that on drop

Reply:
Yeah that is true. I originally thought about having mutable references to constant items which won't work. But for local mutable variables in const fn works fine.